src/app/components/FilterButtons/index.tsx

```diff
@@ -21,7 +22,11 @@ export const FilterButtons = <T extends string>({ options, onSelect, value }: Fi
             clickable
             color="secondary"
             label={
-              <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+              <Box
+                component="span"
+                sx={{ display: 'flex', alignItems: 'baseline', justifyContent: 'center', gap: 2 }}
+              >
+                {selected && <CheckIcon fontSize={'small'} sx={{ transform: 'translateY(4px)' }} />}
```

Comment:
erc721 looks misaligned inside the chip
<img width="212" height="55" alt="image" src="https://github.com/user-attachments/assets/b95e2d95-2d2b-4429-846b-e55aebc4c6af" />

Reply:
Indeed. Fixed. (See the updated screenshot above.)